plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/impl/editor/AbstractEditor.java

```diff
@@ -153,32 +156,57 @@ public void minimize() {
      */
     @Override
     public ContentAssistant openContentAssistant() {
+    	return openContentAssistant(ContentAssistantEnum.DEFAULT);
```

Comment:
Sometimes content assist does not have child items. For example in case of editing pom.xml file, there is only Edit->Content Assist without child items. You need to add logic to handle this case too.